cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -119,20 +119,20 @@ <h3 >
     {% if category and subcategories %}
         <div class="block {% if not page.answer_base.next_step %}block__flush-top{% endif %}">
             <div class="m-related-links">
-                <h2 class="header-slug">
-                    <span class="header-slug_inner">
+                <header class="m-slug-header">
+                    <h2 class="a-heading">
                         Related {{category.name}} subjects
-                    </span>
-                </h2>
-                <ul class="list list__unstyled list__links">
+                    </h2>
+                </header>
+                <ul class="m-list m-list__unstyled m-list__links">
```

Comment:
Oops, nope. Fixed in https://github.com/cfpb/cfgov-refresh/pull/3070/commits/e4fe8e8167d88f551b7d07ca5c008b6e426f7b90